group_membership/variables.tf

```
@@ -0,0 +1,12 @@
+
+variable "memberships" {
+  description = "Map of memberships"
+  type = map(object({
+    group_name   = string
+    user_name    = string
+    access_level = string
+    parent       = string
```

[suggestion] clarify what `parent` means (at the cost of verbosity). I think path is right because it should be like `"top_level/subgroup"` if we're adding to `"top_level/subgroup/sub_subgroup"` right?

```suggestion
    parent_group_path       = string
```